src/s1reader/s1_orbit.py

```diff
@@ -15,20 +16,24 @@
 # Orbital period of Sentinel-1 in seconds:
 # 12 days * 86400.0 seconds/day, divided into 175 orbits
 T_ORBIT = (12 * 86400.0) / 175.0
+PADDING_SHORT = 60
 
 # Temporal margin to apply to the start time of a frame
 #  to make sure that the ascending node crossing is
 #    included when choosing the orbit file
-margin_start_time = datetime.timedelta(seconds=T_ORBIT + 60.0)
+margin_start_time = datetime.timedelta(seconds=T_ORBIT + PADDING_SHORT)
 
 # Scihub guest credential
 scihub_user = 'gnssguest'
 scihub_password = 'gnssguest'
 
 
-def download_orbit(safe_file: str, orbit_dir: str):
+def retrieve_orbit_file(safe_file: str, orbit_dir: str, concatenate: bool=False):
```

Comment:
let's make sure this returns a file list that's sorted by start time

Reply:
It will return the list of the orbit files in ascending order:
```python
    resorb_dict_list = [resorb_dict_earlier, resorb_dict_later]

    if not all(resorb_dict_list):
        warnings.warn('Cannot find RESORB pair that covers the timeframe in the query')
        return

    # Either concatenate the orbit, or return the RESORB pair
    resorb_file_list = []
    for resorb_dict in resorb_dict_list:
        resorb_file = os.path.join(orbit_dir, f"{resorb_dict['orbit_name']}.EOF")
        .
        .
        .
        resorb_file_list.append(resorb_file)
    .
    .
    .
    return resorb_file_list
```